content/applications/sales/sales/products_prices/ewallets_giftcards.rst

```diff
@@ -0,0 +1,143 @@
+:abbr:`PoS (Point of Sale)`. Top-up values can be of different amounts.
+
+.. example::
+   A $50 top-up can be bought for $50, and adds that same amount of credits to the eWallet.
+
+To create a top-up product, go to :menuselection:`Sales app --> Products --> Products` and
+:guilabel:`Create` a new product. On the product template, configure the options as follows:
+
+- :guilabel:`Product Name`: enter a name for the top-up product (for example, `$50 Top-Up`)
+- :guilabel:`Can be Sold`: enabled
+- :guilabel:`Product Type`: select :guilabel:`Service`
+- :guilabel:`Invoicing Policy`: select :guilabel:`Prepaid/Fixed Price`
+- :guilabel:`Create on Order`: select :guilabel:`Nothing`
+- :guilabel:`Sales Price`: enter the amount of the top-up
+
+.. note::
+   In order to have eWallet top-ups of different amounts, create multiple top-up products and
+   modify the :guilabel:`Sales Price` accordingly.
+
+Once the top-up is created, go to :menuselection:`Sales app --> Products --> Gift cards & eWallet`
+to :guilabel:`Create` an eWallet program. The following configuration options are available:
+
+- :guilabel:`Program Name`: enter a name for the eWallet program
+- :guilabel:`Program Type`: select :guilabel:`eWallet`
+- :guilabel:`eWallet Products`: select the eWallet top-up created earlier. Repeat the process if
+  you created top-ups of different amounts.
+- :guilabel:`Email template`: select the email template used for the email sent to the customer. To
+  create a new template, click on the field, select :guilabel:`Search More`, and then click
+  :guilabel:`Create`.
+- :guilabel:`Currency`: select the currency to use for the eWallet program
+- :guilabel:`Company`: select the company for which the program is valid and available
+- :guilabel:`Available On`: select the applications on which the program is valid and available
+- :guilabel:`Website`: select the website on which the program is valid and available. Leave this
+  field empty to include all websites.
+- :guilabel:`Point of Sale`: select the :abbr:`PoS (Point of Sale)` in which the program is valid
+  and available. Leave this field empty to include all :abbr:`PoS (Point of Sale)`.
+
+.. image:: ewallets_giftcards/ewallet-configuration.png
+   :align: center
+   :alt: eWallet program configuration page
+
+Once the program is configured, click the :guilabel:`Generate eWallet` button in the upper-left
+corner to generate eWallets. eWallets can be generated based on :guilabel:`Customers` and/or
+:guilabel:`Customer Tags`. The quantity is automatically adapted according to the
+:guilabel:`Customers` and :guilabel:`Customer Tags` selected. Then, set the :guilabel:`eWallet
+value`. Finally, set the :guilabel:`Valid Until` period if applicable.
+
+Generated eWallets can be accessed through the :guilabel:`eWallets` smart button in the upper-right
+corner. From there, :guilabel:`Send` or :guilabel:`Share` the eWallets via email or a URL link.
+
+.. image:: ewallets_giftcards/ewallet-share.png
```

Comment:
[Compressing](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#images) this image reduces file size by ~2x.